### ISSUE TITLE
Cannot compute top package's path of a native namespace package

### ISSUE THREAD

**Issue by pawamoy** [pawamoy/pytkdocs#19]
**Describe the bug**
Native namespace packages (without `__init__.py` modules) trigger the following kind of error when attempting to retrieve the top package's path:

```
ERROR   -  mkdocstrings.handlers.python: Collection failed: <module 'bot' (namespace)> is a built-in module
Traceback (most recent call last):
  File "/home/dadyarri/projects/bjorn/venv_bjorn/lib/python3.8/site-packages/pytkdocs/cli.py", line 179, in main
    print(json.dumps(process_json(line)))
  File "/home/dadyarri/projects/bjorn/venv_bjorn/lib/python3.8/site-packages/pytkdocs/cli.py", line 116, in process_json
    return process_config(json.loads(json_input))
  File "/home/dadyarri/projects/bjorn/venv_bjorn/lib/python3.8/site-packages/pytkdocs/cli.py", line 98, in process_config
    serialized_obj = serialize_object(obj)
  File "/home/dadyarri/projects/bjorn/venv_bjorn/lib/python3.8/site-packages/pytkdocs/serializer.py", line 143, in serialize_object
    relative_file_path=obj.relative_file_path,
  File "/home/dadyarri/projects/bjorn/venv_bjorn/lib/python3.8/site-packages/pytkdocs/objects.py", line 185, in relative_file_path
    top_package_path = Path(inspect.getabsfile(top_package)).parent
  File "/usr/lib/python3.8/inspect.py", line 720, in getabsfile
    _filename = getsourcefile(object) or getfile(object)
  File "/usr/lib/python3.8/inspect.py", line 696, in getsourcefile
    filename = getfile(object)
  File "/usr/lib/python3.8/inspect.py", line 659, in getfile
    raise TypeError('{!r} is a built-in module'.format(object))
TypeError: <module 'bot' (namespace)> is a built-in module
```

**To Reproduce**
Simply try to collect documentation for a native namespace package.

**Expected behavior**
No exception. We should find another way to determine the top package's path in that case.

Reference: https://github.com/pawamoy/mkdocstrings/issues/79#issuecomment-611492209

**Comment by shyamd** [comment]
I have a solution based on iterating from the top path down till we find a non-namespace package and using that, but it won't work well if the namespace package is further in. Making a test for this is going to be a giant pain. 

Solution:
``` python
subspace_count = len(self.path.split("."))
namespaces = [".".join(self.path.split(".", maxsplit=maxsplit)[:-1]) for maxsplit in range(1, subspace_count)]

for namespace in namespaces:
    try:
        top_package = sys.modules[namespace]
    except KeyError:
        try:
            importlib.import_module(namespace)
        except ImportError:
            return ""
        top_package = sys.modules[namespace]
    try:
        top_package_path = Path(inspect.getabsfile(top_package)).parent
        return str(Path(self.file_path).relative_to(top_package_path.parent))
    except ValueError:
        return ""
    except TypeError:
        pass
```

**Comment by pawamoy** [comment]
Fixed by @shyamd in #22 :heart:
